sh/tests/solutions/left.sh

```diff
@@ -0,0 +1 @@
+wc -l <facts
```

Comment:
As a general information, the student solution can pass without using the redirection

Reply:
i added a restriction